src/styles/components/_Logo.scss

```diff
@@ -0,0 +1,61 @@
+.Logo {
```

Comment:
I think we should use prefixes with this being a pattern library - .logo is fairly common and could cause unwanted conflicts